CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # Changelog
 
+#### Version 1.17.0 (2018-05-30)
+
+New Features:
+ - Add support for gzip compression levels
+   ([#1044](https://github.com/Shopify/sarama/pull/1044)).
+ - Add DeleteRecords operation to Broker
```

Comment:
In general I try and unify/collect/reword PRs to make the changelog a little easier to follow. In this case I'd probably collect the three brand new request/response pairs into something like
```
- Add support for CreatePartitions, DeleteRecords, and DeleteGroups request/response pairs (..., ..., ...)
```